source/bsondump-compatibility-and-installation.txt

```diff
@@ -0,0 +1,60 @@
+.. _bsondump-compatibility-and-installation:
+
+===========================================
+``bsondump`` Compatibility and Installation
+===========================================
+
```

Comment:
Have you updated the TOC so that these pages are findable there? I can't currently find where they "live" in this library.